src/layout/header/Header.tsx

```diff
@@ -107,16 +109,18 @@ export const Header = () => {
             <>
               {customer?.shortName &&
                 (user?.isEditor ? (
-                  <Button
-                    sx={{ whiteSpace: 'nowrap', borderRadius: '2rem' }}
+                  <MenuButton
+                    sx={{
+                      fontSize: '1.25rem',
+                      fontWeight: 700,
+                      textTransform: 'none',
+                    }}
+                    isSelected={currentPath.startsWith(UrlPathTemplate.Editor)}
                     color="inherit"
-                    variant="outlined"
-                    size="small"
-                    component={RouterLink}
                     data-testid={dataTestId.header.editorLink}
                     to={UrlPathTemplate.Editor}>
                     {customer.shortName}
-                  </Button>
+                  </MenuButton>
                 ) : (
                   <Typography variant="button" sx={{ whiteSpace: 'nowrap', color: 'inherit' }}>
```

Comment:
kom det en merge-feil her tro? 🤔 
```suggestion
                  <Typography variant="h1" component="span" sx={{ whiteSpace: 'nowrap', color: 'inherit' }}>
```

Reply:
Aksepterte visst feil versjon her. Fixed :)